
The usual errno and linux-2.6 fixes.

  - Rene Rebe <rene@exactcode.de>

--- dosfstools-2.9/dosfsck/boot.c.vanilla	2005-01-21 23:33:02.259093152 +0100
+++ dosfstools-2.9/dosfsck/boot.c	2005-01-21 23:32:30.262957304 +0100
@@ -9,7 +9,9 @@
 #include <string.h>
 #include <sys/types.h>
 #define _LINUX_STRING_H_
-#include <linux/msdos_fs.h>
+/*#include <linux/msdos_fs.h>*/
+
+#define MSDOS_FAT12 4084 /* maximum number of clusters in a 12 bit FAT */
 
 #include "common.h"
 #include "dosfsck.h"
--- dosfstools-2.9/mkdosfs/mkdosfs.c.vanilla	2005-01-21 23:30:10.329230464 +0100
+++ dosfstools-2.9/mkdosfs/mkdosfs.c	2005-01-21 23:30:26.648749520 +0100
@@ -64,6 +64,7 @@
 #include <sys/types.h>
 #include <unistd.h>
 #include <time.h>
+#include <errno.h>
 
 #if __BYTE_ORDER == __BIG_ENDIAN
 
